packages/next/src/generators/application/schema.json

```diff
@@ -121,9 +121,9 @@
     },
     "appDir": {
       "type": "boolean",
-      "default": false,
-      "description": "Enable experimental app directory for the project",
-      "x-prompt": "Do you want to use experimental app/ in this project?"
+      "default": true,
+      "description": "Enable the App Router for this project.",
+      "x-prompt": "Would you like to use the App Router (recommended)?"
```

Comment:
Should this be removed?

Reply:
I think we still want it just because next.js also prompts for it. And I think there are still bugs with the App Router, like with emotion, even though they say it's "stable".